VideoAPI/VideoApi.AcceptanceTests/Steps/ConferenceSteps.cs

```diff
@@ -278,9 +322,9 @@ private void CreateMultipleConferences(bool addDuplicateFirstNames)
             }
```

Comment:
Hi @valentinabennett There is still no assertion that the conferences that you are expecting are retrieved?

Reply:
The hearing in the past could not be created, so I have check that all known hearings that not meet conditions are not included